### ISSUE TITLE
LF1 accessibility issues

### ISSUE THREAD

**Issue by AbbyANoble** [TeamBasedInquiryLearning/library#338]
[Activity 3.1.15](https://library.tbil.org/2024/precalculus/LF1.html#LR-5-Compare-Slopes-of-Lines)  and [Activity 3.1.17](https://library.tbil.org/2024/precalculus/LF1.html#Direction-and-slope) refer to the graphed lines by their colors. This could be an issue in a printed black and white version or if someone has colorblindness. I'd suggest labeling the graphs as f(x) and g(x) and referring to them that way (with perhaps the color in parentheses for emphasis).
